### PR TITLE
DE45442 - Discussions - Fixing posts attached files broken link

### PR DESCRIPTION
**Pull request by amauryperez**
image

### CODE REVIEW

File: components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js [hunk 1]
```diff
@@ -165,11 +165,11 @@ export class ConsistentEvaluationDiscussionEvidenceBody extends RtlMixin(Localiz
 	_renderAttachments() {
 		if (this.attachmentsList) {
 			return html`${this.attachmentsList.map((attachment) => {
-				const { name, size, extension, href, downloadHref } = attachment.properties;
```
Comment (amauryperez) [inline components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js:168]: downloadHref does not exist for Discussions posts, href is the correct property to use here

**Review by JamieTran — APPROVED** [review]

File: components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js (hunk 1 above)
Comment (JamieTran) [reply components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js:168]: Is this something that changed or was it always `href` 🤔

**Review by tdevos-d2l — APPROVED** [review]

File: components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js (hunk 1 above)
Comment (amauryperez) [reply components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js:168]: I think the discussions API has not been touched in a long time

Comment (amauryperez) [reply components/left-panel/discussions/consistent-evaluation-discussion-evidence-body.js:168]: Looks like it was me here https://github.com/BrightspaceHypermediaComponents/consistent-evaluation/pull/591. That is why I should not touch the front-end :-) . That changed was for assignment and somehow the discussion file was changed as well

**Pull request merged**